src/components/ui/ResponsiveImage.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/require-default-props */
+import PropTypes from 'prop-types'
+
+const ResponsiveImage = ({ 
+  webpSrc, 
+  pngSrc, 
+  alt = 'Michigan DevFest 2025 event logo', 
+  className = '', 
+  ...props 
+}) => {
+  return (
+    <picture>
+      <source srcSet={webpSrc} type="image/webp" />
+      <img 
+        src={pngSrc} 
+        alt={alt || 'Michigan DevFest 2025 event logo'} 
```

Comment:
NIT: Alt is defaulted to the same text so the `||` isn't really needed - Same end result, I'm nitpicking 🤷‍♂️